src/ashell/jerry-code.c

```diff
@@ -326,8 +326,17 @@ void restore_zjs_api() {
 #ifdef BUILD_MODULE_SENSOR
     zjs_sensor_init();
 #endif
+    jerry_value_t global_obj = jerry_get_global_object();
```

Comment:
This is what is done in main.c right now.  I'm fine with moving it to modules_init, but that's for another patch.